src/ripple/rpc/impl/AccountFromString.cpp

```diff
@@ -29,13 +29,11 @@ namespace RPC {
 //
 // Returns a Json::objectValue, containing error information if there was one.
 Json::Value accountFromString (
-    Ledger::ref lrLedger,
     RippleAddress& naAccount,
     bool& bIndex,
```

Comment:
I believe that it is safe to also remove `bIndex` and `iIndex` (use `0`) from the argument list.